Weather/ForecastIO/weather.15m.py

```diff
@@ -201,6 +207,10 @@ def render_wx():
 
   print '---'
 
+  if 'week' in weather_data:
+    print "\n".join(textwrap.wrap(weather_data['week'], 50))
+    print '---'
+
   if 'next_hour' in weather_data:
```

Comment:
Can you move this block to line 203, above the image and add another line break? This information is more relevant versus the weekly summary so it should naturally flow as you're reading